[BUG] inputmask build error

- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

When running e.g. `nx run cookbook:build:production` with inputmask version 5.0.9 we have begun to see the following errors:

```
Error: Module not found: Error: Package path ./lib/extensions/inputmask.numeric.extensions is not exported from package

Error: Module not found: Error: Package path ./lib/inputmask is not exported from package 
```

This is probably related to changed exports section of the inputmask package.json released in 5.0.9: https://robinherbots.github.io/Inputmask/#/changelog

## Which Kirby version was used?
11.2.2

## Are there any additional context?
<!-- Replace this paragraph with any additional context about the problem here (if any) -->

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

--- a/libs/designsystem/form-field/src/directives/date/date-input.directive.ts
+++ b/libs/designsystem/form-field/src/directives/date/date-input.directive.ts
@@ -9,7 +9,7 @@ import {
   LOCALE_ID,
   Renderer2,
 } from '@angular/core';
-import Inputmask from 'inputmask';
+import Inputmask from 'inputmask/dist/inputmask.es6.js';
 
 @Directive({
   standalone: true,

--- a/libs/designsystem/form-field/src/directives/date/date-input.directive.ts
+++ b/libs/designsystem/form-field/src/directives/date/date-input.directive.ts
@@ -9,8 +9,7 @@ import {
   LOCALE_ID,
   Renderer2,
 } from '@angular/core';
-import 'inputmask/lib/extensions/inputmask.date.extensions';
-import Inputmask from 'inputmask/lib/inputmask';
+import Inputmask from 'inputmask';
 
 @Directive({
   standalone: true,

--- a/libs/designsystem/form-field/src/directives/decimal-mask/decimal-mask.directive.ts
+++ b/libs/designsystem/form-field/src/directives/decimal-mask/decimal-mask.directive.ts
@@ -9,8 +9,7 @@ import {
   OnInit,
 } from '@angular/core';
 import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
-import 'inputmask/lib/extensions/inputmask.numeric.extensions';
-import Inputmask from 'inputmask/lib/inputmask';
+import Inputmask from 'inputmask';
 
 interface InputMask {
   unmaskedvalue: () => string;

--- a/libs/designsystem/form-field/src/directives/decimal-mask/decimal-mask.directive.ts
+++ b/libs/designsystem/form-field/src/directives/decimal-mask/decimal-mask.directive.ts
@@ -9,7 +9,7 @@ import {
   OnInit,
 } from '@angular/core';
 import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
-import Inputmask from 'inputmask';
+import Inputmask from 'inputmask/dist/inputmask.es6.js';
 
 interface InputMask {
   unmaskedvalue: () => string;

--- a/libs/designsystem/package.json
+++ b/libs/designsystem/package.json
@@ -34,7 +34,7 @@
     "chartjs-plugin-datalabels": "^2.0.0",
     "date-fns": "^4.1.0",
     "date-fns-tz": "^3.2.0",
-    "inputmask": "5.0.8",
+    "inputmask": "^5.0.9",
     "rxjs": "^7.0.0",
     "swiper": "^9.2.0",
     "zone.js": "^0.14.3 || ~0.15.0"

